pkg/status/toolchaincluster_test.go

```diff
@@ -42,7 +43,7 @@ func TestGetToolchainClusterConditions(t *testing.T) {
 			err := ValidateComponentConditionReady(conditions...)
 
 			// then
-			assert.NoError(t, err)
+			require.NoError(t, err)
```

Comment:
may I know why there are all these changes for replacing `assert` with `require`?

Reply:
that was a feedback from ["testifylint"](https://golangci-lint.run/usage/linters/#testifylint):
```
require-error: for error assertions use require (testifylint)
```
If it's an issue for us, then we could probably try to disable this check 🤷‍♂️

Reply:
ok, that makes sense. It's fine - let's keep it there :+1: